src/Composer/ComposerPackage.php

```diff
@@ -124,13 +123,11 @@ public function __construct(Composer $composer, array $overrideAutoload = null)
             $this->relativePath = $this->packageName;
             $this->packageAbsolutePath = realpath($vendorDirectory . DIRECTORY_SEPARATOR . $this->packageName) . DIRECTORY_SEPARATOR;
         // If the package is symlinked, the path will be outside the working directory.
-        } elseif (0 !== strpos($absolutePath, getcwd()) && 1 === preg_match('/.*\/([^\/]*\/[^\/]*)\/[^\/]*/', $vendorDirectory, $output_array)) {
+        } elseif (0 !== strpos($absolutePath, getcwd()) && 1 === preg_match('/.*[\/\\\\]([^\/\\\\]*[\/\\\\][^\/\\\\]*)[\/\\\\][^\/\\\\]*/', $vendorDirectory, $output_array)) {
             $this->relativePath = $output_array[1];
-        } elseif (1 === preg_match('/.*\/([^\/]*\/[^\/]*)\/composer.json/', $composerJsonFileAbsolute, $output_array)) {
-            // Not every package gets installed to a folder matching its name (crewlabs/unsplash).
+        } elseif (1 === preg_match('/.*[\/\\\\]([^\/\\\\]+[\/\\\\][^\/\\\\]+)[\/\\\\]composer.json/', $composerJsonFileAbsolute, $output_array)) {
```

Comment:
@BrianHenryIE Only now I realised why the relative paths were not created correctly: the regexes only worked for forward slash paths. Here is the fix to make it work with backslash paths also.